frontend/components/forms/validators/valid_url/valid_url.ts

```diff
@@ -6,8 +6,16 @@ interface IValidUrl {
   url: string;
   /**  Validate protocols specified */
   protocols?: ("http" | "https")[];
+  allowAnyLocalHost?: boolean;
 }
 
-export default ({ url, protocols }: IValidUrl): boolean => {
-  return isURL(url, { protocols });
```

Comment:
Wasn't actually validating that the provided url uses one of the provided protocols